test/unittests/test_http.py

```diff
@@ -4,7 +4,9 @@
 import requests
 from urllib3.exceptions import ProtocolError
 from requests.exceptions import ChunkedEncodingError, ConnectionError
-
+from aioresponses import aioresponses
```

Comment:
Can we try using unittest.mock.AsyncMock instead of third party module aioresponses?

Reply:
Removed aioresponses